tpp-app/src/services/cycle/CycleService.js

```diff
@@ -0,0 +1,279 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import {FLOW_LEVEL} from '../utils/constants';
+import {initializeEmptyYear, getDateString, GETsymptomsForDate}  from '../utils/helpers';
+import {Symptoms} from '../utils/models';
+import differenceInCalendarDays from 'date-fns/differenceInDays'
+import isSameDay from 'date-fns/isSameDay'
+import Keys from '../utils/keys';
+
+
+
+/**
+ * Computes the number of days between the two dates provided. If earlierDate and laterDate are equal, returns 1.
+ * @param {Date} earlierDate
+ * @param {Date} laterDate
+ * @return {number} number of days between the two dates provided, ignoring their hours, minutes and seconds.
+ */
+const getDaysDiff = (earlierDate, laterDate) => {
```

Comment:
i would maybe put this in utils since it does involve AsyncStorage and other places may need as well

Reply:
I wasn't sure if I should do that since it's not really computing the difference in days. So if you put in "Jan 21" and "Jan 22" it returns 2. 

I was worried if I put it in helpers people would use it assuming it returns the numeric difference between the date (so "Jan 21" and "Jan 22" returns 1) w/o really reading the docstring. I think a better name would help but I'm not sure what name conveys what the function is doing